### ISSUE TITLE
Project name transfer?

### ISSUE THREAD

**Issue by aireenmei** [jimmydo/grain#3]
Hi! 

Is this repo (pip package "grain") still actively used and maintained?
I would like to publish a project with the name "grain" on PyPI, and wonder if it's possible to request a name transfer from you.

Thanks in advance!

**Comment by jimmydo** [comment]
Hi @aireenmei,

Sorry for the very late response. This repo isn't maintained, so I'd be happy to transfer the PyPI name to you, if you're still interested. Looks like there are 2 options:

1. Add you as an owner of the PyPI project and remove myself from it, or
2. Delete the PyPI project, which will allow you or anyone else to use the name:
   <img width="798" alt="image" src="https://github.com/jimmydo/grain/assets/7198/10d57fd9-46ce-4568-a491-118d2ef5b18e">

Let me know how you'd like to proceed!

**Comment by aireenmei** [comment]
Hi Jimmy,

Yes, I'm still interested. Thank you! Maybe we can first try option #1. My
PyPI account is aireenmei.

Thanks and best regards,
Aireen

**Comment by jimmydo** [comment]
Done. I'll remove myself once you accept the invite.

**Comment by aireenmei** [comment]
Thanks! I've accepted the invite. Is it ok to delete the previous releases?

**Comment by jimmydo** [comment]
Yes, feel free to delete previous releases. I've removed myself from the project, so it's officially yours. 😄
